test/models/user_test.rb

```diff
@@ -503,6 +503,13 @@ class UserTest < ActiveSupport::TestCase
     assert_not_includes(target, users(:kensyuowata))
   end
 
+  test '#year_end_party' do
```

Comment:
ActiveRecordのscopeはクラスメソッドと等価なので下記の方が良さそうです。

```suggestion
  test '.year_end_party' do
```

Reply:
ご指摘ありがとうございます。`#`はインスタンスメソッドですね🤧以後使い分けを気をつけます。
`.`の記法でテスト名を修正いたしました。